StripeCore/StripeCore.xcodeproj/project.pbxproj

```diff
@@ -99,6 +99,8 @@
 		9DCBC08C182ED76A962961E7 /* NSURLComponents+Stripe.swift in Sources */ = {isa = PBXBuildFile; fileRef = D475CD59778FAC1028670AB5 /* NSURLComponents+Stripe.swift */; };
 		9FBA50345D53E82AA974F672 /* STPAPIClient+FileUpload.swift in Sources */ = {isa = PBXBuildFile; fileRef = F4FC2FC535DAF9B340F9EA75 /* STPAPIClient+FileUpload.swift */; };
 		A03F7C3B2E79CB2B006F0FDC /* AsyncTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = A03F7C3A2E79CB28006F0FDC /* AsyncTests.swift */; };
+		A3DA747D2EB16F6200FFFCC9 /* Timeout.swift in Sources */ = {isa = PBXBuildFile; fileRef = A3DA747C2EB16F6200FFFCC9 /* Timeout.swift */; };
```

Comment:
You can remove these changes, the xcodeproj will now pull in source files automatically